actors/runtime/src/runtime/fvm.rs

```diff
@@ -431,13 +442,13 @@ pub fn trampoline<C: ActorCode>(params: u32) -> u32 {
     let mut rt = FvmRuntime::default();
     // Invoke the method, aborting if the actor returns an errored exit code.
     let ret = C::invoke_method(&mut rt, method, &params)
-        .unwrap_or_else(|err| fvm::vm::abort(err.exit_code() as u32, Some(err.msg())));
+        .unwrap_or_else(|err| fvm::vm::abort(err.exit_code().value(), Some(err.msg())));
 
-    // Abort with "illegal actor" if the actor failed to validate the caller somewhere.
+    // Abort with "unspecified" if the actor failed to validate the caller somewhere.
     // We do this after handling the error, because the actor may have encountered an error before
     // it even could validate the caller.
     if !rt.caller_validated {
-        fvm::vm::abort(ExitCode::SysErrIllegalActor as u32, Some("failed to validate caller"))
+        fvm::vm::abort(ExitCode::USR_UNSPECIFIED.value(), Some("failed to validate caller"))
```

Comment:
This should be the "user assertion failed" one

Reply:
fixed